CONTRIBUTING.md

```diff
@@ -115,15 +115,15 @@ log is formatted as a short title (first line) and an extended description
 The short title is the most important part, as it is what will appear in the
 `shortlog` changelog (one line per commit, so no description shown) or in the
 GitHub interface unless you click the "expand" button. As the name tells it,
-try to keep that first line relatively short (ideally <= 50 chars, though it's
+try to keep that first line relatively short (ideally <= 50 characters, though it's
```

Comment:
Fair enough, but "it's" should be sent to the next line. Otherwise it's a bit weird to have a hard wrapped sentence with 84 characters advising to wrap at 80 chars :)